ludwig/encoders/image_torchvision_encoders.py

```diff
@@ -0,0 +1,740 @@
+import logging
+import os
+from abc import abstractmethod
+from typing import Any, Dict, Optional, Union
+
+import torch
+import torchvision.models as tvm
+
+from ludwig.api_annotations import DeveloperAPI
+from ludwig.constants import IMAGE
+from ludwig.encoders.image_encoders import ImageEncoder
+from ludwig.encoders.registry import register_encoder
+from ludwig.schema.encoders.image_torchvision_encoders import (
+    TVAlexNetEncoderConfig,
+    TVConvNeXtEncoderConfig,
+    TVDenseNetEncoderConfig,
+    TVEfficientNetEncoderConfig,
+    TVGoogLeNetEncoderConfig,
+    TVInceptionV3EncoderConfig,
+    TVMaxVitEncoderConfig,
+    TVMNASNetEncoderConfig,
+    TVMobileNetV2EncoderConfig,
+    TVMobileNetV3EncoderConfig,
+    TVRegNetEncoderConfig,
+    TVResNetEncoderConfig,
+    TVResNeXtEncoderConfig,
+    TVShuffleNetV2EncoderConfig,
+    TVSqueezeNetEncoderConfig,
+    TVSwinTransformerEncoderConfig,
+    TVVGGEncoderConfig,
+    TVViTEncoderConfig,
+    TVWideResNetEncoderConfig,
+)
+from ludwig.utils.image_utils import register_torchvision_model_variants, torchvision_model_registry, TVModelVariant
+
+logger = logging.getLogger(__name__)
+
+
+@DeveloperAPI
+class TVBaseEncoder(ImageEncoder):
```

Comment:
Can we move this file into `image_encoders.py` to keep things consolidated? Otherwise, I would want to add a new directory called `image_encoders` or similar to organize things.

Reply:
`image_torchvision_encoders.py` was created based on [this comment](https://github.com/ludwig-ai/ludwig/pull/2408/files/db08c19a9b719a42d3e88c127775293e07825c58#r1029903929
) from @justinxzhao.